data_processing/jobs/silver/silver-with-publish.yml

```diff
@@ -0,0 +1,548 @@
+# Starter pipeline
```

Comment:
Can you explain what is `silver-with-publish.yml`, `silver-without-AIR.yml` and `silver.yml`? What are the differences? Can you update the comments in each of them at the top to make it easier to understand?

Reply:
@satenderrathee

Reply:
silver-without-AIR.yml is old pipeline ,its not using Independent Runner and silver.yml is the new one using AIR. We can remove silver-with-publish.yml ,its not required.

Reply:
Do we need to keep both old pipeline and the new one?

Reply:
I thinks its better to keep both for now.  silver-without-AIR.yml is not configured as pipeline ,only silver.yml is the active pipeline.

Reply:
silver-with-publish.yml removed.

Reply:
Thanks, and where is the configuration that says which pipelines are active?

Reply:
silver-without-AIR.yml is not configured as pipeline in Azure Devops ,its just there as yml file for reference .No Azure Devops pipeline is using this file. I have also updated its trigger section as trigger:  - none